draft-ietf-jose-hpke-encrypt.md

```diff
@@ -345,11 +429,18 @@ HPKE also offers modes that offer authentication.
 HPKE relies on a source of randomness to be available on the device.
 In Key Agreement with Key Wrapping mode, CEK has to be randomly generated and it MUST be ensured that the guidelines in {{RFC8937}} for random number generations are followed.
 
+## HPKE authentication
+
+Authenticated HPKE modes MUST NOT be used with Key Encryption, as this is trivially insecure.
+
 ## Key Management
 
-Reusing a single KEM key across multiple algorithm combinations MUST be avoided to maintain cryptographic security. Each key and its associated algorithm suite, comprising the KEM, KDF, and AEAD, should be managed independently. This separation prevents unintended interactions or vulnerabilities between suites, ensuring the integrity and security guarantees of each algorithm suite are preserved.
-Additionally, the same key MUST NOT be used for both key wrapping and content encryption, as it may introduce security risks.  It
-creates algorithm confusion, increases the potential for key leakage, cross-suite attacks, and improper handling of the key.
+A single key SHOULD NOT be used as both authentication key and recipient key, as this has not been proven to be safe.
+
+A single key SHOULD NOT be used with both JOSE-HPKE and other algorithms as this might enable cross-protocol attacks.
```

Comment:
Regarding these two, it might be useful to define "use" values for the two roles (e.g., "HPKE" and "HPKE-auth"), in order to signal that a key is only meant for HPKE recipient role, or HPKE sender role.

AFAIK, registered "crv" values for EC2 and OKP key types can represent any currently valid HPKE sender role key pair.